src/SfxWeb/cypress/integration/cluster.spec.js

```diff
@@ -165,6 +165,90 @@ context('Cluster page', () => {
     })
   })
 
+  describe("events", () => {
+    const setup = (fileCluster, fileTasks) => {
+      addRoute('events', fileCluster, apiUrl(`/EventsStore/Cluster/Events?*`))
+      addRoute('repairs', fileTasks, apiUrl('/$/GetRepairTaskList?*'))
+
+      cy.visit('/#/events')
+
+      cy.wait('@getevents')
+      cy.wait('@getrepairs')
+    };
+
+    it("loads properly", () => {
+      addRoute('events', 'empty-list.json', apiUrl(`/EventsStore/Cluster/Events?*`))
+      cy.visit('/#/')
+
+      cy.get('[data-cy=navtabs]').within(() => {
+        cy.contains('events').click();
+      })
+
+      cy.wait('@getevents');
+      cy.url().should('include', 'events');
+
+      cy.get('[data-cy=eventtabs]').within(() => {
+        cy.contains('Cluster')
+        cy.contains('Repair Tasks')
+      })
+    })
+
+    it("repair manager disabled", () => {
+      addRoute('repair-manager-manifest', 'manifestRepairManagerDisabled.json', apiUrl('/$/GetClusterManifest*'))
+
+      cy.visit('/#/events')
+
+      cy.wait('@getrepair-manager-manifest')
+
+      cy.get('[data-cy=eventtabs]').within(() => {
+        cy.contains('Cluster')
+        cy.get('Repair Tasks').should('not.exist')
```

Comment:
assign each tab name to a variable here incase the names ever change it will be easier to update.